pages/enterprise/log_view_widget.rst

```diff
@@ -0,0 +1,121 @@
+###############
+Log View Widget
+###############
+
+********
+Overview
+********
+
+Log View is a widget that presents your log data in a format similar to Common Log Format. 
+In other terms, it has the look and feel of console output. In addition, the Log View widget 
+allows you to scroll through log events and as it populates new lines in real-time.
+
+Of course, the Log View widget will provide you a way to sleuth around your log events, to 
```

Comment:
As a non-native speaker, I had to look up the meaning of `sleuth`. Maybe there is an easier word?

Reply:
Changing "sleuth around" to "investigate".